### PR TITLE
TrailingCommaInMultilineArrayFixer - simplify isMultilineArray condition

### PR DESCRIPTION
**Pull request by TomasVotruba**
Replaces #2975 with 2.2 branch as target

### CODE REVIEW

File: src/Fixer/ArrayNotation/TrailingCommaInMultilineArrayFixer.php [hunk 1]
```diff
@@ -65,12 +65,6 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
      */
     private function fixArray(Tokens $tokens, $index)
```
Comment (keradus) [inline src/Fixer/ArrayNotation/TrailingCommaInMultilineArrayFixer.php]: if you assume now that the array is multiline and don't want to check that statement inside of the method (as it's private and you could assume that), please at least make the function name indicates that

Comment (TomasVotruba) [inline src/Fixer/ArrayNotation/TrailingCommaInMultilineArrayFixer.php]: Sure, how exactly?
`fixMultilineArray`?

**Comment by keradus** [comment]
Thank you @TomasVotruba.

**Comment by TomasVotruba** [comment]
Glad to help

**Pull request merged**